src/main/resources/db/changes/09-create-cart-items-table.yaml

```diff
@@ -0,0 +1,40 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-cart-items-table
+      author: maksymmilian
+      changes:
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false

```

Comment:
For primary/foreign keys and unique columns, It's better to have named constraints